site/en/docs/devtools/workspaces/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: "layouts/doc-post.njk"
```

Comment:
Not exactly sure should I just review the proposed changes or the entire docs. If it's the docs, there are places where we still refer workspace as "Filesystem". And there are still:
- inconsistent capital / lowercase of workspace
- inconsistent plural and singular workspaces